DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -410,27 +417,39 @@ def determine_Syslog_daemon(self):
             return True
         is_Rsyslog_running.log_result_to_file()
         is_Syslog_ng_running.log_result_to_file()
-        self.print_error(self.No_Syslog_daemon_error_message)
         return False
 
     def verify_Syslog_daemon_listening(self):
-        '''
+        """
         Verifying the Syslog daemon is listening on the default 514 port for incoming traffic
-        '''
-        command_name = "verify_Syslog_daemon_listening"
+        """
+        global WARNING_TESTS_COUNT
         command_to_run = "sudo netstat -lnpv | grep " + self.SYSLOG_DAEMON
-        result_keywords_array = [self.SYSLOG_DAEMON, "LISTEN", ":514 "]
-        command_object = BasicCommand(command_name, command_to_run, result_keywords_array)
-        command_object.run_full_test()
-        if command_object.is_successful == "Warn":
-            command_object.print_warning(command_object.command_result_err)
-        elif not command_object.is_successful:
-            command_object.print_warning(self.Syslog_daemon_not_listening_warning)
-
+        result_keywords_array = [self.SYSLOG_DAEMON, ":514 "]
+        command_object = BasicCommand(self.command_name, command_to_run, result_keywords_array)
```

Comment:
I see you removed line 420 so what will the command_name be? (self.command_name is empty?)